### PR TITLE
remove compatible basepath code

### PR DESCRIPTION
**Pull request by v-harihar**
Signed-off-by: Vinayakswami Hariharmath <vharihar@redhat.com>

### CODE REVIEW

File: console/console.go [hunk 1]
```diff
@@ -102,10 +100,6 @@ func GetConsolePluginCR(consolePort int, basePath string, serviceNamespace strin
 	}
 }
 
-func GetBasePath(clusterVersion string) string {
-	if strings.Contains(clusterVersion, "4.11") {
-		return compatibilityBasePath
```
Comment (avanthakkar) [inline console/console.go:107]: Maybe the declaration of var(line 31) should be removed as well?

Comment (v-harihar) [reply console/console.go:107]: removed

**Review by naveenpaul1 — APPROVED** [review]
LGTM

**Comment by naveenpaul1** [comment]
@v-harihar please remove the clusterversions from the bundle folder

**Pull request merged**